### ISSUE TITLE
Unexpected results for DDPM unlearning

### ISSUE THREAD

**Issue by YuanXun2024** [OPTML-Group/Unlearn-Saliency#19]
Dear Authors,

Thanks for releasing the code. I got unexpected results when running the demo instructions for DDPM unlearning.

I ran following instructions in sequence:
(1) CUDA_VISIBLE_DEVICES="0,1" python train.py --config cifar10_train.yml --mode train
(2) CUDA_VISIBLE_DEVICES="0,1" python train.py --config cifar10_saliency_unlearn.yml --ckpt_folder results/cifar10/yyyy_mm_dd_hhmmss --label_to_forget 0 --mode generate_mask
(3) CUDA_VISIBLE_DEVICES="0,1" python train.py --config cifar10_saliency_unlearn.yml --ckpt_folder results/cifar10/yyyy_mm_dd_hhmmss --label_to_forget 0 --mode saliency_unlearn --mask_path results/cifar10/unlearn/mask/{mask_name} --alpha 0.0 --method rl

After running (3), I got the following folder:
![Picture1](https://github.com/user-attachments/assets/4892a550-20c6-4598-8bc6-8df4b0011f8a)

However, the samples in the logs folder look like the following:
![sample-999](https://github.com/user-attachments/assets/424a5355-f773-4046-bfe7-c930ddbdf584)

It looks like all classes are unlearned, which is not the expected result.

Can you please help to figure out the problem? Thanks.

Best,
Yuan Xun

**Comment by a-F1** [comment]
Thank you for your appreciation and recognition of our work. I recommend setting the alpha in command 3 to 1e-3, as outlined in the paper's settings.

In the unlearning process, the hyperparameter α is used to balance the forgetting loss and remaining loss. A smaller α results in stronger forgetting effects. The image you provided shows signs of over-forgetting, where the model fails to generate all types of images. Therefore, it's necessary to increase α accordingly.

I've updated the relevant section in the README.md. If you have any further questions, please feel free to reach out.

**Comment by zhaoyilinnn** [comment]
Thank you. Problem solved.
